runtime/onert/core/src/compiler/train/StaticBackPropShapeInferer.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#ifndef __ONERT_COMPILER_STATIC_DERIVATIVE_SHAPE_INFERER_H__
-#define __ONERT_COMPILER_STATIC_DERIVATIVE_SHAPE_INFERER_H__
+#ifndef __ONERT_COMPILER_STATIC_BACK_PROP_SHAPE_INFERER_H__
+#define __ONERT_COMPILER_STATIC_BACK_PROP_SHAPE_INFERER_H__
```

Comment:
(optional) Strictly speaking, the guard should have `TRAIN` in the middle of the name. But `BACK_PROP` is likely to be used in train only. I am okay. But for consistency, it would be better to have `TRAIN`.